samples_project/Assets/SampleViewer/Samples/3DAttributes/APIMapCreator.cs

```diff
@@ -0,0 +1,256 @@
+// Copyright 2022 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at: http://www.apache.org/licenses/LICENSE-2.0
+//
+
+// ArcGISMapsSDK
+
+// @@Start(namespaces)
```

Comment:
We don't use this type of commenting/tagging anywhere else in the samples. We should probably remove it.